packages/pmml-editor/src/editor/components/EditorScorecard/organisms/CorePropertiesTable.tsx

```diff
@@ -154,7 +153,10 @@ export const CorePropertiesTable = (props: CorePropertiesTableProps) => {
         }
       }}
     >
-      <PageSection variant={PageSectionVariants.light}>
+      <PageSection
+        variant={PageSectionVariants.light}
+        className={`editable-item${isEditModeEnabled ? "--editing" : ""}`}
```

Comment:
```suggestion
        className={isEditModeEnabled ? "editable-item" : "editable-item--editing"}
```

Since CSS classes are not typed in any way, it's better to have it pass the [grep test](http://jamie-wong.com/2013/07/12/grep-test/)